plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/PayloadFactoryMediatorPropertiesEditionPartForm.java

```diff
@@ -981,7 +981,7 @@ public void validate() {
         eu.showEntry(argsTableElements, false);
         
         // Sets a default XML value. If not, parsing to XML fails and is unable to load the source view
-        if (getMediaType().getName().equals(MediaType.XML.getName())) {
+        if (getMediaType().getName().equals(MediaType.XML.getName()) && getPayload().equals("")) {
```

Comment:
Can getPayload() become null?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/1063